# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/streams/UnderlyingSinkCallbackHelpers.h
# Commit: 7d5683180ac3
# Full Hash: 7d5683180ac3f9e7cc96d68dde27b8b48642a7ac
# Author: Yulia Startsev <ystartsev@mozilla.com>
# Date: 2022-02-18 09:08:22
# Regressor Bug: 1742437
# File Overlap Count: 5
# Description:
#   Bug 1742437 - re-scope all generic module/script loader files under js/loader; r=jonco
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D138133
# ==============================================================================

diff -r a48aff5f2555 -r 7d5683180ac3 dom/streams/UnderlyingSinkCallbackHelpers.h
--- a/dom/streams/UnderlyingSinkCallbackHelpers.h	Thu Feb 17 15:57:52 2022 +0000
+++ b/dom/streams/UnderlyingSinkCallbackHelpers.h	Thu Feb 17 15:57:52 2022 +0000
@@ -7,8 +7,8 @@
 #ifndef mozilla_dom_UnderlyingSinkCallbackHelpers_h
 #define mozilla_dom_UnderlyingSinkCallbackHelpers_h
 
+#include "js/loader/ModuleMapKey.h"
 #include "mozilla/HoldDropJSObjects.h"
-#include "mozilla/dom/ModuleMapKey.h"
 #include "mozilla/dom/Promise.h"
 #include "mozilla/dom/UnderlyingSinkBinding.h"
 #include "nsISupports.h"